bin/queuePopulator.js

```diff
@@ -63,14 +63,11 @@ async.waterfall([
             probeServer.addHandler([DEFAULT_LIVE_ROUTE, DEFAULT_READY_ROUTE],
                 (res, log) => queuePopulator.handleLiveness(res, log)
             );
-            // TODO: set this variable during deployment
             // enable metrics route only when it is enabled
-            if (process.env.ENABLE_METRICS_PROBE === 'true') {
-                probeServer.addHandler(
-                    DEFAULT_METRICS_ROUTE,
-                    (res, log) => queuePopulator.handleMetrics(res, log)
-                );
-            }
+            probeServer.addHandler(
```

Comment:
We can remove the comment above now that we enable this by default

Reply:
good catch, will do